src/rsz/src/RecoverPower.cc

```diff
@@ -98,6 +98,7 @@ bool RecoverPower::recoverPower(const float recover_power_percent, bool verbose)
 
   int end_index = 0;
   int failed_move_threshold = 0;
+  IncrementalParasiticsGuard guard(resizer_);
```

Comment:
Was the previous code incorrect?  I don't see any removal that this corresponds to.

Reply:
Yes, same for RepairSetup (I believe the incr. parasitics only got enabled by accident through journalEnd/journalRevert)